src/main/java/org/spongepowered/common/mixin/core/data/MixinCustomDataHolder.java

```diff
@@ -107,9 +108,9 @@ public boolean hasManipulators() {
     @Override
     public List<DataManipulator<?, ?>> getCustomManipulators() {
         final List<DataManipulator<?, ?>> list = Lists.newArrayList();
-        for (DataManipulator<?, ?> manipulator : this.manipulators) {
-            list.add(manipulator.copy());
-        }
+        list.addAll(this.manipulators.stream()
+                .map(DataManipulator::copy)
```

Comment:
Mixins do NOT properly remap anonymous classes and will fail to apply at runtime due to the Mixin class being referenced but not available at runtime.